src/content/docs/de/guides/integrations-guide/svelte.mdx

```diff
@@ -0,0 +1,158 @@
+import Since from '~/components/Since.astro';
+
+Diese **[Astro-Integration][astro-integration]** erlaubt dir serverseitiges Rendering und clientseitige Hydratation von deinen [Svelte-Komponenten](https://svelte.dev). Sie unterstützt Svelte 3, 4 und 5 (experimentell).
+
+## Installation
+
+Astro verfügt über einen `astro add`-Befehl, der das Setup offizieller Integrationen automatisiert. Wenn du möchtest, kannst du die [Integrationen auch manuell installieren](#manuelle-installation).
+
+Um `@astrojs/svelte` zu installieren, führe einen der folgenden Befehle in einem neuen Terminal-Fenster aus.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```sh
+  npx astro add svelte
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```sh
+  pnpm astro add svelte
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```sh
+  yarn astro add svelte
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Sollten dir dabei Probleme begegnen, [melde sie gerne bei uns auf GitHub](https://github.com/withastro/astro/issues) und versuche stattdessen die manuelle Installation.
+
+### Manuelle Installation
+
+Installiere zuerst das `@astrojs/svelte`-Paket:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```sh
+  npm install @astrojs/svelte
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```sh
+  pnpm add @astrojs/svelte
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```sh
+  yarn add @astrojs/svelte
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Die meisten Paketmanager installieren auch die zugehörigen Peer-Abhängigkeiten. Wenn du jedoch beim Starten von Astro die Warnung "Cannot find package 'svelte'" (oder eine ähnliche Meldung) erhältst, musst du Svelte manuell installieren:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```sh
+  npm install svelte
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```sh
+  pnpm add svelte
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```sh
+  yarn add svelte
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Wende dann die Integration auf die Datei `astro.config.*` an, indem du die Eigenschaft `integrations` verwendest:
+
+```js title="astro.config.mjs" ins={2} ins="svelte()"
+import { defineConfig } from 'astro/config';
+import svelte from '@astrojs/svelte';
+
+export default defineConfig({
+  // ...
+  integrations: [svelte()],
+});
+```
+
+## Erste Schritte
+
+Um deine ersten Svelte-Komponenten in Astro zu verwenden, gehe zu unserer [UI-Framework Dokumentation][astro-ui-frameworks]. Dort lernst du:
+
+* 📦 wie Framework-Komponenten geladen werden,
+* 💧 clientseitige Hydratationsoptionen, und
+* 🤝 verschiedene Möglichkeiten, unterschiedliche Frameworks zu mischen
+
+## Optionen
+
+Diese Integration wird von `@sveltejs/vite-plugin-svelte` unterstützt. Um den Svelte-Compiler anzupassen, können Optionen für die Integration genutzt werden. Weitere Informationen findest du in der [`@sveltejs/vite-plugin-svelte`-Dokumentation](https://github.com/sveltejs/vite-plugin-svelte/blob/HEAD/docs/config.md).
```

Comment:
"powered by" als "unterstützt von" zu übersetzen, scheint mir den Sinn leicht zu verschieben. Mir fällt aber gerade auch nichts Besseres ein und bin auch nicht sicher, ob es überhaupt relevant ist.

Reply:
ich hab echt lange überlegt, wie ich das übersetzen soll. _wird unterstützt von_ war die beste schlechte Variante, die mir eingefallen ist

Reply:
Ich hätte eben gedacht, dass es eher in die Richtung "Diese Integration basiert auf `@sveltejs/vite-plugin-svelte`" gehen müsste. Aber bin überhaupt nicht sicher, wäre gut noch andere Meinungen zu kriegen.

Reply:
```suggestion
Diese Integration wird durch @sveltejs/vite-plugin-svelte ermöglicht. Um den Svelte-Compiler anzupassen, können Optionen für die Integration genutzt werden. Weitere Informationen findest du in der [`@sveltejs/vite-plugin-svelte`-Dokumentation](https://github.com/sveltejs/vite-plugin-svelte/blob/HEAD/docs/config.md).
```
Das hier wär evtl. auch eine Option.

Reply:
Mir gefällt Max' Version mit Abstand am besten!

Reply:
Oh ja, das ist gut! Gekauft!